common/fuzzer_utils.py

```diff
@@ -121,10 +116,10 @@ def validate(fuzzer):
         return False
 
 
-def get_fuzzer_from_config(fuzzer_config: dict) -> str:
+def get_fuzzer_from_config(fuzzer_config: dict) -> Optional[str]:
```

Comment:
I'd like to get rid of the command line version of variants too frankly.
Why can this return None? If it does return None does that break callers?

Reply:
This is just to satisfy linter, it will never return None. This is just return 'fuzzer' attribute which should always exist.

Command line variants are all removed already - https://github.com/google/fuzzbench/pull/452
This fuzzer config is just a way of passing in old code, it is just a dict with one fuzzer attribute [there are no env, name attribute anymore, see they are removed in this cl]. i can remove fuzzer-config entirely and use fuzzer name only, but that should in another cl, this cl is already complicated.

Reply:
It is confusing to me why we even have parts of the code using fuzzer configs even though we don't support them anymore.

Reply:
Sorry didn't see your last comment before I left (the rest of) my review.
I'm fine with doing it in another CL.